.github/workflows/main.yml

```diff
@@ -82,6 +82,14 @@ jobs:
           - 2.7
           - 3.7
           - 3.8
+    # start nginx as service to test validation via http-challenge local path
+    services:
+      nginx:
+        image: nginx
+        ports:
+          - 5002:80
+        options: -v /var/tmp:/usr/share/nginx/html
```

Comment:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idservicesservice_idvolumes

Reply:
reverted back to "volumes" as this parameter works as intended b87bbbb